Tests/CouchDBTests/DocumentBulkUpdateTests.swift

```diff
@@ -0,0 +1,350 @@
+/**
+* Copyright IBM Corporation 2016, 2017
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import XCTest
+
+#if os(Linux)
+	import Glibc
+#else
+	import Darwin
+#endif
+
+import Foundation
+import SwiftyJSON
+
+@testable import CouchDB
+
+class DocumentBulkUpdateTests: XCTestCase {
+
+	// Add an additional class variable holding all tests for Linux compatibility
+	static var allTests: [(String, (DocumentBulkUpdateTests) -> () throws -> Void)] {
+		return [
+			("testBulkInsert", testBulkInsert),
+			("testBulkUpdate", testBulkUpdate),
+			("testBulkDelete", testBulkDelete)
+		]
+	}
+
+	// To enable running Linux and OSX tests in parallel
+#if os(Linux)
+	let dbName = "test_db_linux"
+#else
+	let dbName = "test_db_db"
+#endif
+
+	// MARK: - Database connection properties
+
+	static let dbSecuredConnection = true
```

Comment:
Please set `dbSecuredConnection` to false. Our testing cloudant instance is not setup for https connections.